artiq/frontend/artiq_mkfs.py

```diff
@@ -41,6 +54,10 @@ def main():
         for key, filename in args.f:
             with open(filename, "rb") as fi:
                 write_record(fo, key, fi.read())
+        for action in args.e:
+            if action == "write_ch_names":
+                cntn = Cntn(args.channel_names).get_config_string()
+                write_record(fo, "channel_names", cntn.encode())
```

Comment:
Remove

Reply:
One can just use ``-f`` for this...

Reply:
Or even ``-s $(...)``. Learn how to use the shell.

Reply:
Do you mean I should store the config string into the file directly, instead of processing it again?

Reply:
All your modifications to mkfs and coremgmt are superfluous.